README.md

```diff
@@ -1,2 +1,124 @@
 # apex_launchtest
-Framework for ROS2 Integration Testing
+![build status](https://gitlab.com/ApexAI/apex_rostest/badges/master/build.svg) ![coverage](https://gitlab.com/ApexAI/apex_rostest/badges/master/coverage.svg)
+
+This tool is a framework for ROS2 integration testing using the [ros2 style launch description](https://github.com/ros2/launch/blob/master/ros2launch/examples/example.launch.py).
+It works similarly to rostest, but makes it easier to inspect the processes under test.
+
```

Comment:
Could we maybe list couple things that you can test with this framework, e.g.:
1. the process/node under test started and shutdown correctly
1. two or more nodes under test successfully established ros topic connection
1. the process/node under test died unexpectedly
1. the process/node received the data and published the result on stdout or ros topic
1. the process/node received the right arguments
1. ...